src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/Example1.java

```diff
@@ -0,0 +1,58 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MethodLength">
+      <property name="max" value="4"/>
+    </module>
+  </module>
+</module>
+*/
+
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.sizes.methodlength;
+
+// xdoc section -- start
+public class Example1 {
+
+  // violation below, 'Method Example1 length is 5 lines (max allowed is 4)'
+  public Example1() {
+    int var1 = 2;
+    int var2 = 4;
+    int sum = var1 + var2;
+  }
+
+  public Example1(int a)  {
```

Comment:
Please add empty line or comment `// ok became ....`
In this case all examples will be aligned code, and location of code on line numbers.

Reply:
@romani done

Reply:
![1](https://github.com/checkstyle/checkstyle/assets/125591358/eadfc926-0d07-4528-8a78-d17c587d0cfd)
![2](https://github.com/checkstyle/checkstyle/assets/125591358/2aad05ec-ac91-4412-93f8-fe6217f8b96d)
@romani please review the changes is this ok?